packages/serverless/src/apollo/app/database.ts

```diff
@@ -20,7 +21,8 @@ export type TFindIndexRequest = TDatabaseRequest & {
 
 const DatabaseCreateRequest = Joi.object<TDatabaseCreateRequest>({
   databaseName,
-  merkleHeight: Joi.number().integer().positive().required()
+  merkleHeight: Joi.number().integer().positive().required(),
```

Comment:
```suggestion
  merkleHeight: Joi.number().integer().positive().min(8).max(255).required(),
```

Reply:
Or may be max is 128, would be better than 255.

Reply:
Done